components/vm/12-interpolated-defaults.tf

```diff
@@ -3,12 +3,25 @@ locals {
     "stg"  = "nle"
     "prod" = "prod"
   }
+
+  xdr_env = var.env == "stg" ? "nonprod" : var.env
+
+  xdr_tags_map = {
+    activityName = "heritage"
+    application  = "crimeportal"
+    org          = "hmcts"
+    server       = "server"
+    env          = local.xdr_env
+  }
+
+  xdr_tags = jsonencode(local.xdr_tags_map)
```

Comment:
I'm not sure jsonencode is applicable here as its just expecting a string in the format of `"tag1,tag2,tag3,etc.."`. It will need to be formed like [this](https://github.com/hmcts/oracle-azure-infrastructure/blob/master/terraform/modules/azure-vm/main.tf#L153)